README.md

```diff
@@ -1,90 +1,198 @@
+
+> **Note**
+> The values specified in `c.JupyterHub.template_vars` will overwrite the default values set in `nebari_jupyterhub_theme/templates`
+
+## Using `nebari_jupyterhub_theme` on your JupyterHub instance 📦
+
+1. Install `nebari_jupyterhub_theme`:
+
+   ```bash
+   pip install nebari_jupyterhub_theme
+   ```
+
+   > **Note**
+   > If you already have a JupyterHub configuration file you can jump to the next step. Otherwise, follow the instructions in the [JupyterHub documentation](https://jupyterhub.readthedocs.io/en/stable/getting-started/config-basics.html) to generate a configuration file first.
+
+2. Add the following to you JupyterHub configuration file to pick up the new `jinja2` templates directory and static files:
+
+   ```python
+   from nebari_jupyterhub_theme import theme_extra_handlers, theme_template_paths
+
+   c.JupyterHub.extra_handlers = theme_extra_handlers
+
+   c.JupyterHub.template_paths = theme_template_paths
+   ```
+
+3. **Optional** - if you need to further customize the UI you can modify the `template_vars` within your JupyterHub config file by adding the following lines to it:
+
+   ```python
+   # minimal configuration example overwriting the default values in the nebari_jupyterhub_theme
+   c.JupyterHub.template_vars = {
+    "hub_title": "Welcome to Nebari",
+    "logo": "/hub/custom/images/Nebari-logo-square.svg",
+    "primary_color": "#cb39ed",
+    "secondary_color": "#2bd1c5",
+   ```
+
+The available configuration variables are:
+
+- `hub_title`: The title of the JupyterHub instance. This will be displayed in the header of the login page. (H1 heading, `default: Welcome to Nebari`)
+- `hub_subtitle`: Subtitle shown under the main instance Title. (H2 heading, `default: ""`)
+- `welcome`: Welcome message on the login page (`default:  Welcome to Nebari. For more information about Nebari, visit <a href="https://nebari.dev/">https://nebari.dev</a>`)
+- `logo`: Logo displayed on the navbar (`default: Nebari logomark`)
+- `primary_color`: Primary color (`default: #cb39ed`)
+- `secondary_color`: Secondary color (`default: #2bd1c5`)
+- `accent_color`: Accent color (`default: #eda61d`)
+- `text_color`: Text color (`default: #1c1d26`)
+- `h1_color`: H1 color, main Title in login page (`default: #0f1015`)
+- `h2_color`: H2 color, subtitle in login page (`default: #0f1015`)
+- `navbar_text_color`: Navbar links and text color (`default: #1c1d26`)
+- `navbar_hover_color`: Hover color for navbar links (`default: #00a3b0`)
+
+> **Note**
+> The default colors, typefaces and logos are chosen based on the Nebari branding. You can find more about this on the [Nebari design repository](https://github.com/nebari-dev/nebari-design).
+
+:computer: You can see an example of these variables and the configuration in [`test_jupyterhub_config.py`](test_jupyterhub_config.py).
+
+Nebari internal options:
+- `cdsdashboards_enabled`
```

Comment:
using the `cdsdashboards_enabled` option will require the jupyterhub environment to contain `cdsdashboards` to work correctly.

Reply:
👍🏽  - I was unsure as this is entirely "nebari specific" but you are correct we should add this